pkg/gui.py

```
@@ -10,16 +10,34 @@
+
 from .TIPA import ProcessManager
+from logger_config import logger
+
+
+if os.path.exists("settings.json"):
+    try:
+        with open("settings.json") as settings_file:
+            settings = json.load(settings_file)
+            pytesseract.pytesseract.tesseract_cmd = settings.get(
+                "tesseract_path",
+                r"D:\Program Files\Tesseract-OCR\tesseract.exe"
```

Ruff (0.8.0)

Trailing comma missing

Add trailing comma

(COM812)